osx/make-installer.sh

```diff
@@ -13,9 +13,20 @@ ptb=""
 BUILD_DIR="source/build"
 SOURCE_DIR="source"
 
-# shellcheck disable=SC2154
-# If run via Github Actions, use Qt5_DIR for the custom Qt location - otherwise assume system default
-[ -n "$Qt5_DIR" ] && QT_DIR="${Qt5_DIR}" || QT_DIR="/usr/local/opt/qt/bin"
+# Set HOMEBREW_PREFIX, HOMEBREW_CELLAR, add correct paths to PATH
+ eval "$(brew shellenv)"
+
+
+# If run via Github Actions, use Qt5_DIR for the custom Qt location - otherwise use QT_DIR
```

Comment:
:thinking: As you will have spotted, we have been and are currently using Qt5 for MacOS builds but at some point we will want to at least try and get Qt6 supported - can this script be constructed to handle both cases?

Reply:
Here's a run for that branch if you're curious: 
https://github.com/rparet/Mudlet/actions/runs/11397887377

Can start making qt6 builds by bumping `qt` variable in the build matrix to a qt6 version that's supported by aqt / install-qt. 

Post migration to qt6 there will be some housekeeping / cleanup tasks to remove qt5 specific affordances, but that's to be expected.